commands/previewPush.go

```diff
@@ -216,6 +228,65 @@ DomainLoop:
 		totalCorrections += len(corrections)
 		anyErrors = printOrRunCorrections(domain.Name, domain.RegistrarName, corrections, out, push, interactive, notifier) || anyErrors
 	}
+
+	managedDomains := make([]string, 0, len(cfg.Domains))
+	for _, zone := range cfg.Domains {
+		managedDomains = append(managedDomains, zone.Name)
+	}
+
+	if args.DeleteUnmanagedDomains {
+		fmt.Printf("Checking Domain Removal:\n")
```

Comment:
Suppose a user has a provider that doesn't support `ListZones()` and they provide the` --delete-unmanaged-zones` flag.  They won't see the `Checking Domain Removal:` message.  They might wonder why "the flag isn't working".  Can they get a message that lets them know they've used a flag that isn't supported by their provider?

Reply:
There is a check for that just two lines down.